Gemfile

```diff
@@ -6,10 +6,16 @@ else
   gem 'puppet', :require => false
 end
 
+gem 'mocha'
+gem 'diff-lcs'
+gem 'json_pure'
+gem 'json'
 gem 'metadata-json-lint'
 gem 'puppetlabs_spec_helper', '>= 0.1.0'
 gem 'facter', '>= 1.7.0'
 gem 'rspec-puppet'
+gem 'rake', '~> 10.5.0'
```

Comment:
Rake should be down in the ruby 1.8.7 conditional such as https://github.com/ghoneycutt/puppet-module-ssh/blob/master/Gemfile